### ISSUE TITLE
[Quest] 26290 | Secrets of the Tower - Westfall

### ISSUE THREAD

**Issue by iTz-AddriX** [WoWManiaUK/Blackwing-Lair#4330]
**Links:** https://www.wowhead.com/quest=26290/secrets-of-the-tower

**What is happening:** When you try to drink [Potion of Shrouding](https://www.wowhead.com/item=58112/potion-of-shrouding) it says you must be in Mortwake's Tower, but when there are [Elite Mercenaries](https://www.wowhead.com/npc=42656/mercenary) outside so you cannot enter to drink potion.

**What should happen:** After you drink [Potion of Shrouding](https://www.wowhead.com/item=58112/potion-of-shrouding)(Which should be able to do outside of Tower) you should turn yourself invisible to slip past the [Elite Mercenaries](https://www.wowhead.com/npc=42656/mercenary), and make way to top of Tower to find out what Helix is hiding.

**Comment by Echinaceax** [comment]
@iTz-AddriX have you tried a little closer to tower like in the video ?
https://www.wowhead.com/quest=26290/secrets-of-the-tower#videos:id=84317

**Comment by iTz-AddriX** [comment]
Yes, even tried directly inside. It's like you cannot even drink it at all, and you also use it on the Mercenaries as well makes them invisible but yet can still attack and kill you.

**Comment by Echinaceax** [comment]
ok then its an item issue :)

**Comment by Echinaceax** [comment]
bq next update
